contracts/voting-verifier/src/query.rs

```diff
@@ -1,28 +1,263 @@
+use axelar_wasm_std::voting::PollStatus;
 use connection_router::state::{CrossChainId, Message};
+use cosmwasm_schema::cw_serde;
 use cosmwasm_std::{Deps, StdResult};
 
 use crate::error::ContractError;
-use crate::state::VERIFIED_MESSAGES;
+use crate::state::{self, Poll, POLLS, POLL_MESSAGES};
 
-pub fn verification_statuses(
-    deps: Deps,
-    messages: Vec<Message>,
-) -> StdResult<Vec<(CrossChainId, bool)>> {
+#[cw_serde]
+pub enum VerificationStatus {
+    Verified,
+    FailedToVerify,
```

Comment:
Do we actually need this status? I feel like we could just use `NotVerified` for this.

Reply:
Also confused.

Reply:
I tried to represent different status regardless of some being used same way for some things

Reply:
I had this discussion with @eguajardo, in the end we settled on "define states by what they represent, not what they are used for"